quicktest.go

```diff
@@ -10,6 +10,40 @@ import (
 	"testing"
 )
 
+// Check runs the given check using the provided t and continues execution in
+// case of failure. For instance:
+//
+//     qt.Check(t, answer, qt.Equals, 42)
+//     qt.Check(t, got, qt.IsNil, qt.Commentf("iteration %d", i))
+//
+// Additional args (not consumed by the checker), when provided, are included as
+// comments in the failure output when the check fails.
+func Check(t testing.TB, got interface{}, checker Checker, args ...interface{}) bool {
+	return check(checkParams{
```

Comment:
```
return New(t).Check(t, got, checker, args...)
```
(and for Assert)

?
then you can remove the extra `check` function too.

Reply:
Done.